### DISCUSSION TITLE
Anyone use Upload with react-html5-camera-photo?

### DISCUSSION THREAD

**Discussion by selinosherpa** [rpldy/react-uploady#241] [Q&A] [answered]
I'm about to try to use https://github.com/mabelanger/react-html5-camera-photo with Uploady. I'm wondering if anyone here has already gone through the effort to connect HTML5 camera access to Uploady with success?

**Comment by yoavniran** [comment, marked as answer]
Created a simple example: https://codesandbox.io/s/react-uploady-with-react-html5-camera-photo-rxz00?file=/src/App.js

hopefully can be useful.

**Reply by selinosherpa** [reply]
Fantastic! Thank you, @yoavniran. The work you're doing here is top-notch. 👏🏾
I'll be sure to submit our use case to your "in production" list.

**Reply by yoavniran** [reply]
that'll be awesome! thanks @selinosherpa
